indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -77,7 +77,10 @@ def _process(self):
         form = SessionForm(obj=FormDefaults(colors=self._get_random_color()))
         if form.validate_on_submit():
             create_session(self.event_new, form.data)
-            return jsonify_data(html=_render_session_list(self.event_new))
+            sessions = [{'id': s.id, 'title': s.title, 'colors': s.colors}
+                        for s in self.event_new.sessions if not s.is_deleted]
```

Comment:
i'm not sure whether I made `sessions` a dynamic relationship but if I did you could also use `in self.event_new.sessions.filter_by(is_deleted=False)` - but filtering on the python-side is perfectly fine too as there are rarely more deleted than undeleted sessions (and in any case, the number of sessions per event is low enough to make it completely irrelevant)